CMakeLists.txt

```diff
@@ -581,9 +581,8 @@ if (WIN32)
 	)
 
 	target_compile_definitions(torrent-rasterbar
-		PUBLIC
-			_WIN32_WINNT=0x0600 # target Windows Vista or later
-			WIN32_LEAN_AND_MEAN # prevent winsock1 to be included
+		PRIVATE _WIN32_WINNT=0x0600 # target Windows Vista or later
```

Comment:
@arvidn, please take a look at python bindings build. Now windows compilers say:
```
Please define _WIN32_WINNT or _WIN32_WINDOWS appropriately. For example:
  - add -D_WIN32_WINNT=0x0501 to the compiler command line; or
  - add _WIN32_WINNT=0x0501 to your project's Preprocessor Definitions.
  Assuming _WIN32_WINNT=0x0501 (i.e. Windows XP target).
```
https://ci.appveyor.com/project/arvidn/libtorrent/builds/19573355/job/tlq5rsq1qvapefg2#L1700

Reply:
right, I think the python bindings and the tests are fine to be built against the XP version of the runtime. They don't use any fancy windows stuff anyway. Does the warning bother you?
It would be easy enough to add this define to the bindings and tests builds.